wpiutil/src/main/native/fmtlib/src/os.cpp

```diff
@@ -277,7 +277,6 @@ std::size_t file::read(void* buffer, std::size_t count) {
 std::size_t file::write(const void* buffer, std::size_t count) {
   rwresult result = 0;
   FMT_RETRY(result, FMT_POSIX_CALL(write(fd_, buffer, convert_rwcount(count))));
-  if (result < 0) FMT_THROW(system_error(errno, "cannot write to file"));
   return detail::to_unsigned(result);
```

Comment:
This one seems like it could be problematic.  If result is -1, it's going to return UINT_MAX?  It might be better to return count in the error case?

Reply:
Why count as opposed to zero or something?

Reply:
That’s what write() returns on success (the number of bytes written).  So if you have an outer loop you may be incrementing based on the return value, so there’s a potential for an infinite loop if this always returns 0.